.kuzzlerc.sample

```diff
@@ -778,6 +778,11 @@
           }
         }
       },
+      // Global settings to apply by default (can be overriden by user requests)
+      "globalSettings": {
```

Comment:
Maybe `defaultSettings` or `defaultShardSettings` would be better

Reply:
I like defaultSettings